test/sql/storage/catalog/test_storage_sequences.cpp

```diff
@@ -19,8 +19,12 @@ TEST_CASE("Use sequences over different runs without checkpointing", "[storage]"
 		REQUIRE_NO_FAIL(con.Query("CREATE SEQUENCE seq_cycle INCREMENT 1 MAXVALUE 3 START 2 CYCLE;"));
 		result = con.Query("SELECT nextval('seq')");
```

Comment:
Some more sequence tests to copy and use for currval:

test/sql/alter/add_col/test_add_col_default_seq.test
test/sql/storage/wal/wal_store_default_sequence.test
test/sql/storage/catalog/test_store_default_sequence.test

Reply:
A whole lot of tests have been added. Happy to add more if required.